react/src/components/atoms/forms/InputNumber/index.js

```diff
@@ -100,29 +89,31 @@ const NumberInput = (props) => {
           };
 
           const handleAdjust = (e) => {
+            e.persist();
             let direction;
             if (e.currentTarget === upRef.current) {
               direction = 'up';
             } else {
               direction = 'down';
             }
             const inputEl = ref.current;
-            if (direction === 'up' && (!hasProperty(props, 'max') || inputEl.value < props.max)) {
-              if (is.empty(inputEl.value)) {
-                inputEl.value = 1;
-              } else {
-                inputEl.value = Number(numbro(inputEl.value)
-                  .add(props.step).value());
-              }
-              handleChange(e);
-            } else if (direction === 'down' && (!hasProperty(props, 'min') || inputEl.value > props.min)) {
-              if (is.empty(inputEl.value)) {
-                inputEl.value = -1;
-              } else {
-                inputEl.value = Number(numbro(inputEl.value)
-                  .subtract(props.step).value());
-              }
-              handleChange(e);
+            let newValue = inputEl.value ? Number(inputEl.value) : inputEl.value;
+            if (direction === 'up' && (!hasNumberProperty(props, 'max') || newValue < props.max)) {
+              newValue = newValue ? (newValue + props.step).toFixed(countDecimals(props.step)) : props.step;
```

Comment:
if newValue is a string, toFixed won't work because it doesn't exist.
```suggestion
              newValue = newValue ? Number(newValue + props.step).toFixed(countDecimals(props.step)) : props.step;
```

Reply:
newValue is always a value, see [line 100](https://github.com/massgov/mayflower/pull/518/files#diff-6eb9a368ca77d1f2523e456949ebae6fR100)

Reply:
Don't this this is neccessary —> newValue will always be a number value based on line 100 and line 102.

Reply:
this check is not needed, you can only enter numbers, periods, commas, or null in an input type number, so this will be a number, null, or a string of . or , - null and strings will evaluate to false above which is correct so this is fine.